lib/model/app_model/list_invoices.dart

```diff
@@ -70,7 +70,17 @@ class AppInvoice {
         "NEW RESPONSE HERE : ${req.amount} ${req.paymentRequest} ${req.prefix} ${req.signature} ${req.tags} ${req.timestamp}");
     var amount = req.amount;
     var paymentRequest = req.paymentRequest;
-    var prefix = req.prefix;
+    var pre = req.prefix;
+    String prefix = '';
+    if (pre == pay_req.PayRequestPrefix.lntb) {
+      prefix = "lntb";
+    } else if (pre == pay_req.PayRequestPrefix.lnbcrt) {
+      prefix = "lncbrt";
+    } else if (pre == pay_req.PayRequestPrefix.lnbc) {
+      prefix = "lnbc";
+    } else {
+      prefix = "lnsb";
+    }
```

Comment:
you should document what this if else is doing!

Reply:
Oh yes yes, I will update it.